src/corelibs/U2Gui/src/util/U2FileDialog.cpp

```diff
@@ -33,6 +33,59 @@
 
 namespace U2 {
 
+
+namespace {
```

Comment:
Use static on the function level, not a static namespace. 
Reason: the reader can't see the scope in 2-3 pages away from the function, so using 'static' is a required style for UGENE. This is a blocker.

Reply:
OK